frontend/packages/console-shared/src/components/modal/Modal.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+import { Modal as PfModal, ModalProps as PfModalProps } from '@patternfly/react-core';
+
+type ModalProps = { isFullScreen?: boolean } & PfModalProps & any;
```

Comment:
used `any` here because while using PfModalProp getting this typeerror
```
Type '{ appendTo: () => HTMLElement; children: ReactNode; className?: string; isOpen?: boolean; header?: ReactNode; title?: string; 'aria-labelledby'?: string; 'aria-label'?: string; ... 365 more ...; ouiaSafe?: boolean; }' is not assignable to type 'IntrinsicClassAttributes<Modal>'.
  Types of property 'ref' are incompatible.
    Type 'LegacyRef<HTMLDivElement>' is not assignable to type 'LegacyRef<Modal>'.
      Type '(instance: HTMLDivElement) => void' is not assignable to type 'LegacyRef<Modal>'.
        Type '(instance: HTMLDivElement) => void' is not assignable to type '(instance: Modal) => void'.
          Types of parameters 'instance' and 'instance' are incompatible.
            Type 'Modal' is missing the following properties from type 'HTMLDivElement': align, addEventListener, removeEventListener, accessKey, and 234 more.
```

Reply:
Thanks for the fix. seems to work fine. Updated.